### PR TITLE
Sync SDK with latest API changes

### PR DESCRIPTION
**Pull request by ctumolosus**

### CODE REVIEW

File: docs/organizations.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ const organizations = client().organizations;
 | id | string | Filter by one or more IDs of this resource |
 
 
+##`getSettingsAndCurrenciesByOrganizationId(organizationId, options)`
```
Comment (ctumolosus) [inline docs/organizations.md:61]: 😮

Comment (chrhicks) [reply docs/organizations.md:61]: yeah :( @ericluria !

**Comment by chrhicks** [comment]
lgtm

File: docs/organizations.md (hunk 1 above)
Comment (ericluria) [reply docs/organizations.md:61]: I blame api-build! Naming the parameter `id` caused linting errors.

**Pull request merged**